Can't build Plugin_DotScene

<!--
If you have a question rather than reporting a bug please go to
- https://gitter.im/OGRECave/ogre or
- https://forums.ogre3d.org/
where you get much faster responses.

This is a template helping you to create an issue which can be processed as quickly as possible.
-->
#### System Information
- Ogre Version: **1.12.3** latest master (revision e9ddd1a07fe7d5ea123f6ed8192b47f3d53da537)
- Operating System / Platform: **Ubuntu 18.04.3 LTS** 
- RenderSystem: **none**

#### Detailled description
<!-- your description -->
I cloned repo, create **cmake-build-debug** directory and run

```bash
cmake ..
make Plugin_DotScene -j 4
```

And got error:

[ 92%] Building CXX object Samples/CMakeFiles/DefaultSamples.dir/Common/src/DefaultSamplesPlugin.cpp.o
/usr/bin/ld: ../../Dependencies/lib/libpugixml.a(pugixml.cpp.o): relocation R_X86_64_PC32 against symbol `_ZN4pugi16xml_parse_resultC1Ev' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status
PlugIns/DotScene/CMakeFiles/Plugin_DotScene.dir/build.make:87: recipe for target 'lib/Plugin_DotScene.so.1.12.3' failed
make[2]: *** [lib/Plugin_DotScene.so.1.12.3] Error 1
CMakeFiles/Makefile2:782: recipe for target 'PlugIns/DotScene/CMakeFiles/Plugin_DotScene.dir/all' failed
make[1]: *** [PlugIns/DotScene/CMakeFiles/Plugin_DotScene.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 93%] Linking CXX shared library ../lib/DefaultSamples.so
[ 99%] Built target DefaultSamples
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2


#### Ogre.log
none

--- a/CMake/Dependencies.cmake
+++ b/CMake/Dependencies.cmake
@@ -144,15 +144,16 @@ if(OGRE_BUILD_DEPENDENCIES AND NOT EXISTS ${OGREDEPS_PATH})
 
     message(STATUS "Building pugixml")
     file(DOWNLOAD
-        https://github.com/zeux/pugixml/releases/download/v1.9/pugixml-1.9.tar.gz
-        ${PROJECT_BINARY_DIR}/pugixml-1.9.tar.gz)
+        https://github.com/zeux/pugixml/releases/download/v1.10/pugixml-1.10.tar.gz
+        ${PROJECT_BINARY_DIR}/pugixml-1.10.tar.gz)
     execute_process(COMMAND ${CMAKE_COMMAND}
-        -E tar xf pugixml-1.9.tar.gz WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
+        -E tar xf pugixml-1.10.tar.gz WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
     execute_process(COMMAND ${BUILD_COMMAND_COMMON}
-        ${PROJECT_BINARY_DIR}/pugixml-1.9
-        WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/pugixml-1.9)
+        -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE # this will be linked into a shared lib
+        ${PROJECT_BINARY_DIR}/pugixml-1.10
+        WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/pugixml-1.10)
     execute_process(COMMAND ${CMAKE_COMMAND}
-        --build ${PROJECT_BINARY_DIR}/pugixml-1.9 ${BUILD_COMMAND_OPTS})
+        --build ${PROJECT_BINARY_DIR}/pugixml-1.10 ${BUILD_COMMAND_OPTS})
 
     find_package(Freetype)
     if (NOT FREETYPE_FOUND)

